entity-test-lib/src/main/java/org/terracotta/passthrough/PassthroughClusterControl.java

```diff
@@ -18,27 +18,35 @@
  */
 package org.terracotta.passthrough;
 
+import java.net.URI;
+
 import org.terracotta.connection.Connection;
+import org.terracotta.connection.ConnectionException;
+import org.terracotta.connection.ConnectionFactory;
 
 
 /**
  * The implementation used to control the passthrough testing cluster.
  */
 public class PassthroughClusterControl implements IClusterControl {
+  private final String stripeName;
   // We track the "original" server state for tear-down.
   private final PassthroughServer originalActiveServer;
   private final PassthroughServer originalPassiveServer;
 
   // The active we are currently using can change on restart.
   private PassthroughServer activeServer;
 
-  public PassthroughClusterControl(PassthroughServer activeServer, PassthroughServer passiveServer) {
+  public PassthroughClusterControl(String stripeName, PassthroughServer activeServer, PassthroughServer passiveServer) {
```

Comment:
Actually we are using it to create another Connection to the same stripe to simulate another client. But we could have just used the ConnectionService with the right scheme. So the answer is: yes, but I'll change that :-)